bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/FieldType.java

```diff
@@ -38,7 +38,9 @@ public enum FieldType {
     IMAGE("image"),
     TELEPHONE("tel"),
     PASSWORD("password"),
-    RANGE("range");
+    RANGE("range"),
```

Comment:
Is this part of the spec as well ?

Reply:
If you mean HTML spec, then yes: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input

Reply:
And if you mean AF spec, here is the PR in af2-docs: https://git.corp.adobe.com/livecycle/af2-docs/pull/213

Reply:
Please bump af2 spec here in the PR as well for completion

Reply:
The af2 spec PR needs to be merged and released for it to work.